apps/medusa-be/.env.template

```
@@ -26,6 +26,8 @@ STORE_CORS=http://localhost:8000,http://localhost:9000,http://localhost
 ADMIN_CORS=http://localhost:5173,http://localhost:9000
 AUTH_CORS=http://localhost:5173,http://localhost:9000
 MEDUSA_BACKEND_URL=localhost
+# Public storefront base URL (used to build links in transactional emails)
+STOREFRONT_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STOREFRONT_URL key should go before the STORE_CORS key

(UnorderedKey)